pkgs/applications/networking/instant-messengers/discord/default.nix

```diff
@@ -49,7 +59,7 @@ let
     downloadPage = "https://discordapp.com/download";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
-    maintainers = with maintainers; [ MP2E artturin infinidoge ];
+    maintainers = with maintainers; [ MP2E artturin infinidoge jopejoe1 ];
```

Comment:
This should be separated into its own commit

Reply:
done